docs/reports/reports.rst

```diff
@@ -1,6 +1,230 @@
-.. vale off
-
 Reports
 #######
 
-.. vale on
\ No newline at end of file
+Mautic's Reports menu allows you to generate highly customizable Reports.
+
+You can use the reporting tools to track marketing metrics, identify marketing activities that are effective or need improvement, and troubleshoot or investigate if you are curious about why something is happening.
+
+To get to Reports, click the Reports link from the menu on the left side of your instance. To create a new Report, click the **+New** button in the top right corner.
+
+.. image:: images/mautic-report.png
+   :align: center
+   :alt: Screenshots of Mautic Report
+
+|
+
+
+Data sources
+============
+
+The **Details** tab on a Report contains the same options across all Reports and provides some general options for your Report.
+
+.. image:: images/mautic-report-details.png
+   :align: center
+   :alt: Screenshots of Mautic Report Details
+
+* **Name** - Specify a Report name that makes it easy for you and other Users to identify the purpose of the Report.
+
+* **Data source** - Select the data source appropriate to the Report that you are building. Note that each data source has a different set of available columns, filters, and graphs. The available data sources are:
+
+1. :doc:`Assets</components/assets>`
+
+   * Assets
+   * Asset Downloads
+
+2. :doc:`Campaigns</campaigns/campaigns_overview>`
+   
+   * Campaign Events
+
+3. :doc:`Channels Messages</queue/message_queue>`
+   
+   * Message Queue
+
+4. :doc:`Emails</channels/emails>`
+    
+   * Emails
+   * Emails Sent
+
+5. :doc:`Forms</components/forms>`
+   
+   * Forms
+   * Form Submissions
+
+6. :doc:`Contacts</contacts/manage_contacts>`
+   
+   * Contacts
+   * Multi Touch Attributions
+   * First Touch Attributions
+   * Last Touch Attributions
+   * Contact Point Log
+   * Frequency Rules
+   * Segment Membership
+   * Do Not Contact
+   * UTM Codes
+
+7. :doc:`Companies</companies/companies_overview>`
+   
+   * Companies
+
+8. :doc:`Mobile Notifications</channels/marketing_messages>`
+   
+   * Mobile Notifications
+   * Mobile Notifications Sent
+
+9. :doc:`Pages</components/landing_pages>`
+    
+   * Landing Pages
+   * Page hits
+
+10. Videos
+
+    * Video hits
+
+As demonstrated, Mautic displays the data sources in the format: <Parent data source> <Child data source>
```

Comment:
```suggestion
As demonstrated, Mautic displays the data sources in the format: ``<Parent data source> <Child data source>``
```

I think this should be inlined to make it more visible? What do you think?

Reply:
I agree